oe_contact_forms.install

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * @file
+ * The OpenEuropa Contact Forms installation file.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Implements hook_install().
+ */
+function oe_contact_forms_install() {
+  // Configure the RDF SKOS graphs.
+  if (!\Drupal::service('config.installer')->isSyncing()) {
```

Comment:
Indeed insipred from oe_content, event though its only one graph, I used the service for the configuration sync check. In addition to what oe_content offers I add to existing graphs instead of overwriting.

Reply:
https://github.com/openeuropa/rdf_skos/pull/46